android/src/ti/imagefactory/ImageFactoryModule.java

```diff
@@ -148,89 +162,142 @@ private TiBlob imageTransform(int type, TiBlob blob, KrollDict args)
 		image.recycle();
 		image = null;
 		ref = null;
-		
+
 		return result;
 	}
 
- 	// Public Image Methods
+	@Kroll.method
+	public TiBlob fixOrientation(TiBlob blob) {
```

Comment:
I think we should rename the method `fixOrientation()` to `imageAsUpright()`.

This way it follows the same naming convention as the rest of this module's methods and the method name better indicates what it does (in my opinion).